frontend/src/metabase/dashboard/hooks/use-dashboard-url-params.ts

```diff
@@ -87,9 +92,27 @@ export const useDashboardUrlParams = ({
     titled,
     font,
     setFont,
-    theme,
+    theme: normalizedTheme,
```

Comment:
I've tried passing `normalizedTheme` to `useLocationSync`, but it didn't work because we can't sync values that has been normalized since the actual could have changed, but the normalized value is still the same. e.g. changing from `theme=light` to `theme=transparent` results in `normalizedTheme: "light"` the same.

What we should do is to keep syncing the original value, but normalizing it when using.